src/main/java/org/gridsuite/filter/utils/expertfilter/FieldType.java

```diff
@@ -115,7 +115,6 @@ public enum FieldType {
     SUBSTATION_PROPERTIES,
     SUBSTATION_PROPERTIES_1,
     SUBSTATION_PROPERTIES_2,
-    SUBSTATION_PROPERTIES_3,
```

Comment:
Even if this value is obsolete we might have to keep it (with a comment explaining why) because otherwise it is now impossible to get inside a folder containing a filter using SUBSTATION_PROPERTIES_3. Because of the following crash on filter-server : 

```
java.lang.IllegalArgumentException: No enum constant org.gridsuite.filter.utils.expertfilter.FieldType.SUBSTATION_PROPERTIES_3
	at java.base/java.lang.Enum.valueOf(Enum.java:273) ~[na:na]
	at org.hibernate.type.descriptor.java.EnumJavaType.fromName(EnumJavaType.java:254) ~[hibernate-core-6.5.2.Final.jar:6.5.2.Final]
```